### ISSUE TITLE
Unit tests break server startup by removing private directory

### ISSUE THREAD

**Issue by haganbmj** [dekkerglen/CubeCobra#642]
**Describe the bug**
Unit tests blow away the `/private` directory to perform card database tests. This means that after each test run a new card database must be downloaded in order to start the server.

**To Reproduce**
1. Run `npm test` or more specifically `npm test -t updatecards`
2. Observe the deletion of `/private`
3. `npm start`

```
SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at ReadFileContext.callback (C:\git\CubeCobra\serverjs\cards.js:74:30)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:437:13)
[nodemon] app crashed - waiting for file changes before starting...
```

**Expected behavior**
Unit tests should not interfere with stored files, instead a new directory should be constructed and used specifically for tests if file system writes are a required part of testing.
